apps/storefront/.env.template

```diff
@@ -26,3 +26,7 @@ ORYX_CONTENTFUL_SPACE=
 
 # The Storyblok Access Token
 ORYX_STORYBLOK_TOKEN=
+
+# Used to set a preset, for now if it's not empty then b2b preset will be loaded.
+# If it's empty then storefront preset will be loaded.
```

Comment:
I'd add a list of available options. Also not empty is not intuitive, i.e. ORYX_PRESET=B2C should not result in B2B, what do you think? Maybe we should add bit more logic to make it already work as intended?

Reply:
I wanted to keep it as simple as possible, but you're right. In any case we will need to make it at some point.